grammar.ebnf

```diff
@@ -20,8 +20,19 @@ StructDecl = "struct", Identifier, "{", { StructField }, "}";
 StructField = Identifier, ":", Type, ";";
 
 
+Statement = "{", { Statement }, "}"
+          | Type, Identifier, ";"
```

Comment:
What's this?

Reply:
It's a block statement. It opens up a new scope and can be used in conjunction with if and while statements. Although now I see that my code in while loops obligates braces whilst the if does not.

I'll also make it so that the variable definition works with multiple variables per type.

Reply:
I pushed a new commit making the if and while consistent. If you believe it's better to have mandatory braces, I'll add them.

Reply:
> It's a block statement.

I'm referring to line 24. It seems like a variable _declaration_. Which is... a declaration, not a statement :). We may allow "declare anywhere" but for the time being, we'll just have all variable declarations at the start.

Reply:
Oh fair enough. You marked both lines so I assumed you meant both.

Reply:
Actually I didn't and as I said [here](https://github.com/ACM-UoA-Student-Chapter/dil/pull/16#discussion_r352233741) I don't know of a way to make Github mark only one specific line.